metricbeat/module/ceph/mgr_osd_pool_stats/_meta/data.json

```diff
@@ -0,0 +1,27 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "ceph": {
+        "mgr_osd_pool_stats": {
```

Comment:
Same here, would it make sense to use `osd_pool_stats`?

Reply:
Answered above.